app/models/like.rb

```diff
@@ -0,0 +1,6 @@
+class Like < ActiveRecord::Base
+
+  belongs_to :user
+  belongs_to :picture
+  validates :user, uniqueness: { scope: :picture, message: "has liked this picture already" }
```

Comment:
Metrics/LineLength: Line is too long. [93/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)